packages/app/src/embed/components/Content/index.tsx

```diff
@@ -387,7 +387,12 @@ export default class Content extends React.PureComponent<Props, State> {
     } = this.props;
 
     const mainModule = isInProjectView
-      ? findMainModule(sandbox.modules, sandbox.directories, sandbox.entry)
+      ? findMainModule(
+          sandbox.modules,
+          sandbox.directories,
+          sandbox.entry,
+          sandbox.template
```

Comment:
`Notes for reviewer`: Not sure what Content does with this information, but keeping it consistent with App